pi5/board.go

```diff
@@ -41,44 +41,45 @@ func RegisterBoard(modelName string, gpioMappings map[string]gl.GPIOBoardMapping
 	resource.RegisterComponent(
 		board.API,
 		Model,
-		resource.Registration[board.Board, *Config]{
+		resource.Registration[board.Board, *resource.NoNativeConfig]{
```

Comment:
sorry you will have to change this back because the change I merged yesterday for pull up/pull down now has a config